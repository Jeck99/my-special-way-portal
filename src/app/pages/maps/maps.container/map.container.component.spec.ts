jest.mock('@angular/router');

import { MapContainerComponent } from './map.container.component';

describe('MapContainerComponent', () => {});
