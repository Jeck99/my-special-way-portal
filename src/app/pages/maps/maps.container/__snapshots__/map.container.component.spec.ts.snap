// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapContainerComponent should match snapshot 1`] = `
<app-maps-container
  availableFloors={[Function Array]}
  currentFloor="0"
  currentFloorLocations={[Function Array]}
  locationService={[Function Object]}
  locations={[Function Array]}
>
  <app-map-view />
</app-maps-container>
`;
